cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -220,6 +221,12 @@ func StartControllers(s *options.CloudControllerManagerServer, kubeconfig *restc
 	nodeController.Run()
 	time.Sleep(wait.Jitter(s.ControllerStartInterval.Duration, ControllerStartJitter))
 
+	// Start the PersistentVolumeLabelController
+	pvlController := pvlcontroller.NewPersistentVolumeLabelController(client("pvl-controller"), cloud)
+	threads := 5
```

Comment:
No real reason other than a previous commenter on this controller didn't like that the original controller wasn't threaded.  This seems to be a fairly common construct for controllers, but none of the other controllers that I looked at explained why 5 threads was ideal for their cases either.  Processing 5 volumes at a time seems reasonable and possibly overkill imo.